roles/cobbler/templates/kickstarts/cephlab_ubuntu.preseed

```diff
@@ -3,6 +3,12 @@
 # Fetch the os_version from the distro using this profile.
 #set os_version = $getVar('os_version','')
 
+# Fetch Ubuntu version (e.g., 14.04)
+#set distro = $getVar('distro','').split("-")[1]
+
```

Comment:
Good call.  `distro` is already a variable defined in each profile.

I changed `distro` to `distro_ver` and `distro_ver` to `distro_ver_major`.